content/en/tracing/setup_overview/custom_instrumentation/dotnet.md

```diff
@@ -119,9 +119,30 @@ using (var parentScope =
     }
 }
 ```
+
+### Headers extraction and injection
```

Comment:
I'm not sure this information belongs here in the custom instrumentation page. I think once we add the recently-discussed API for injecting/extracting headers, we should include code snippets of that here. And then we can link over to the "setup" page for the reader to learn more about configuring the headers style

Reply:
Thanks. I put it here as it was there in Java and Go as well. 
But indeed, next week, I'll push the new API and document it here. And I can only link this part. I'll see.

Reply:
Fixed in 42303f0b25c378a5d52566507547057fba1bb8bf